src/inference.c

```diff
@@ -59,9 +59,10 @@ void resolve(Onnx__ModelProto *model,
     // More than 1 opset can be imported. Iterate n_opset_import
```

Comment:
This comment should be on line 54
`all_context[nodeIdx].outputs[i]->data_type = 1;`

The whole point of this patch was to remove this hardcoding, so lets remove than line and TODOs. This output type is now being properly setted with the preparer functions.

Reply:
will do